src/components/10-atoms/text/index.js

```diff
@@ -16,9 +15,7 @@ class AXAText extends NoShadowDOM {
   }
 
   static get styles() {
-    return css`
-      ${unsafeCSS(styles)}
-    `;
+    return styles;
```

Comment:
shouldn't this be changed on all components then?

Reply:
for example cookie-disclaimer has `${unsafeCSS(styles)}`

Reply:
Yes, it should. But that's another issue, I opened https://github.com/axa-ch/patterns-library/issues/2006.